numpy/__init__.py

```diff
@@ -182,9 +185,40 @@
     oldnumeric = 'removed'
     numarray = 'removed'
 
-    # We don't actually use this ourselves anymore, but I'm not 100% sure that
-    # no-one else in the world is using it (though I hope not)
-    from .testing import Tester
+    if sys.version_info[:2] >= (3, 9):
```

Comment:
Using `__getattr__` is a pretty neat trick.

 Isn't the version check here supposed to be `>= (3, 7)` though?

Reply:
Yeah. It should. Sorry. This was left over from my testing. I was trying to figure out what was happening with the builtins